pkg/controller/datavolume-controller.go

```diff
@@ -169,10 +170,18 @@ const (
 	NamespaceTransferInProgress = "NamespaceTransferInProgress"
 	// MessageNamespaceTransferInProgress is a const for reporting target transfer
 	MessageNamespaceTransferInProgress = "Transferring PersistentVolumeClaim for DataVolume %s/%s"
+	// ImgSizeNotReady is a const to report that it's not yet possible to obtain the size of the image
+	ImgSizeNotReady = "Image size not ready"
 
 	// AnnCSICloneRequest annotation associates object with CSI Clone Request
 	AnnCSICloneRequest = "cdi.kubevirt.io/CSICloneRequest"
 
+	// AnnVirtualImageSize annotation contains the Virtual Image size of a PVC used for host-assisted cloning
+	AnnVirtualImageSize = "cdi.Kubervirt.io/virtualSize"
```

Comment:
why "SourceCapacity" starts with capital letter and "virtualSize" not? Do we have some rules about that?

Reply:
My fault, I didn't notice that. I'll change "VirtualSize" to lowercase.